LOAN/LoansSpecific/LoanProducts.rdf

```diff
@@ -55,74 +57,84 @@
 	xmlns:sm="http://www.omg.org/techprocess/ab/SpecificationMetadata/"
 	xmlns:xsd="http://www.w3.org/2001/XMLSchema#">
 	
-	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/LoanProducts/">
+	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/LOAN/LoansSpecific/LoanProducts/">
```

Comment:
I think most of this can be deleted. 
It misses product-related aspects such as target market including demographics and required credit scoring

Reply:
@rivettp - Let's address products in another pass - not worth holding up this pull request for, and needs an SME.